src/App.js

```diff
@@ -1,12 +1,12 @@
-import React, { Component } from "react";
-import "./App.css";
-import Input from "./components/input/index";
+import React from "react";
+import Form from "./components/Form";
 
-class App extends Component {
+class App extends React.Component {
```

Comment:
Why was `React.` added?

Reply:
ESlint complained about { Component }. Could have been a bug, changed back and nothing from it.